DPGAnalysis/Skims/test/test_TopMuEG_SKIM.py

```diff
@@ -550,6 +539,7 @@
         'keep TotemVFATStatusedmDetSetVector_ctppsDiamondRawToDigi_*_*',
         'keep CTPPSDiamondRecHitedmDetSetVector_ctppsDiamondRecHits_*_*',
         'keep CTPPSDiamondLocalTrackedmDetSetVector_ctppsDiamondLocalTracks_*_*',
+        'keep TotemTimingLocalTrackedmDetSetVector_diamondSampicLocalTracks_*_*',
```

Comment:
The following keep statements were present in the master version of this PR, but missing here:
```PYTHON
        'keep TotemFEDInfos_totemT2Digis_*_*',
        'keep TotemT2DigiedmNewDetSetVector_totemT2Digis_*_*',
        'keep TotemVFATStatusedmDetSetVector_totemT2Digis_*_*',
```
Since this is a test script it does not prevent getting merged, but probably it is something you want to have a look at, @soureek